src/pages/team.js

```diff
@@ -1,25 +1,115 @@
+import Navbar from "../components/navbar"
+import Officer from "../components/officer"
 import data from '../data/team.json'
+import "../styles/team.css"
 
-const officers = data.officers;
-officers.map(officer => console.log(officer))
+class Team extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            activeTeam: 'Administrative'
+        }
+    }
 
+    handleTeamChange = (_, { value }) => {
+        this.setState({ activeTeam: value })
+    }
 
-const Team = () => (
-   <div>
-   <p>team page</p>
-   {officers.map(officer => 
-      <Card textAlign="center" key={officer.email}>
-         <Image src={officer.img} circular />
-         <Card.Content>
-         <Card.Header>{`${officer.name} '${officer.year}`}</Card.Header> 
-         <Card.Meta>{officer.team}</Card.Meta>
-         </Card.Content>
-      </Card>
-      )}
-   </div>
-)
+    render() {
+        const officers = data.officers.filter(officer => officer.team.includes(this.state.activeTeam))
+        const teamOptions = [
+            {
+                key: 'Administrative',
+                text: 'Administrative',
+                value: 'Administrative',
+            },
+            {
+                key: 'Design',
+                text: 'Design',
+                value: 'Design',
+            },
+            {
+                key: 'Social',
+                text: 'Social',
+                value: 'Social',
+            },
+            {
+                key: 'Corporate',
+                text: 'Corporate',
+                value: 'Corporate',
+            },
+            {
+                key: 'Academic',
+                text: 'Academic',
+                value: 'Academic',
+            },
+            {
+                key: 'Publicity',
+                text: 'Publicity',
+                value: 'Publicity',
+            },
+            {
+                key: 'Webdev',
+                text: 'Webdev',
+                value: 'Webdev',
+            },
+            {
+                key: 'Alumni',
+                text: 'Alumni',
+                value: 'Alumni',
+            },
+        ]
+        
+        return (
+            <div>
+                <Navbar activePage="team" />
+                <Container fluid>
+                    <Header className="team-header">
+                        Meet the{' '}
+                        <Dropdown
+                            inline
+                            options={teamOptions}
+                            defaultValue={teamOptions[0].value}
+                            onChange={this.handleTeamChange}
+                        />
+                        {' '}Team
+                    </Header>
+                    <Container>
+                        <Header as="h2" className="officers-header">Team Leads</Header>
+                        <div className="officers-container">
+                            {
+                                officers.filter(officer => {
```

Comment:
could just add a field in the json for all team leads + eboard so you can just check for that field instead of checking for these